client/landing/stepper/declarative-flow/link-in-bio.ts

```diff
@@ -43,7 +43,11 @@ export const linkInBio: Flow = {
 					return navigate( 'linkInBioSetup' );
 
 				case 'linkInBioSetup':
-					return window.location.replace( '/start/link-in-bio/domains' );
+					return window.location.replace(
+						`/start/newsletter/domains?new=${ encodeURIComponent(
```

Comment:
This sends LiB users to newsletter flow in /start. Fixed in https://github.com/Automattic/wp-calypso/pull/66973.

Reply:
aghh
thanks for addressing it